src/app.scss

```diff
@@ -6,14 +6,19 @@
 }
 
 .App {
+  .text-areas {
```

Comment:
This is duplicated, probably some auto-merge trickery here -- probably safe to leave this here if we want to merge asap tho

Reply:
haha, whoops. yep, I got it removed